### PR TITLE
Reverse avatar colors.

### PR DESCRIPTION
**Pull request by burakukula**
Before
screen shot 2016-12-13 at 17 12 46

After
screen shot 2016-12-13 at 17 12 17

### CODE REVIEW

File: src/components/avatar/avatar.html [hunk 1]
```diff
@@ -55,42 +55,42 @@ <h3 class="docs-block__header">Default avatars with border!</h3>
...
-          <svg class="sg-icon sg-icon--x94 sg-icon--gray-secondary">
+          <svg class="sg-icon sg-icon--x94 sg-icon--gray">
             <use xlink:href="#icon-profile"></use>
           </svg>
         </div>
       </div>
       <div class="sg-avatar sg-avatar--xxxlarge sg-avatar--with-border">
         <div class="sg-avatar__image sg-avatar__image--icon">
-          <svg class="sg-icon sg-icon--x118 sg-icon--gray-secondary">
+          <svg class="sg-icon sg-icon--x118 sg-icon--gray">
```
Comment (kdzwinel) [inline src/components/avatar/avatar.html]: I hope you haven't cheated using `cmd+r` and changed all of these by hand !!

**Comment by aju** [comment]
@burakukula did you agree with Greg that we can change it everywhere like that?

**Comment by burakukula** [comment]
@aju yes, Greg did agree to this change. and @kdzwinel yes, I did it by hand. I was pure pleasure.

File: src/components/avatar/avatar.html (hunk 1 above)
Comment (aju) [inline src/components/avatar/avatar.html]: There is more places that needs this change! :)

Comment (burakukula) [inline src/components/avatar/avatar.html]: @aju please have a look

**Pull request merged**